sdk/textanalytics/azure-ai-textanalytics-perf/src/main/java/com/azure/ai/textanalytics/perf/ServiceTest.java

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.textanalytics.perf;
+
+import com.azure.ai.textanalytics.TextAnalyticsAsyncClient;
+import com.azure.ai.textanalytics.TextAnalyticsClient;
+import com.azure.ai.textanalytics.TextAnalyticsClientBuilder;
+import com.azure.core.credential.AzureKeyCredential;
+import com.azure.core.util.Configuration;
+import com.azure.perf.test.core.PerfStressTest;
+
+/**
+ * The Text Analytics service base class to setup the performance tests.
+ * @param <TOptions> The options type.
+ */
+public abstract class ServiceTest<TOptions> extends PerfStressTest<TextAnalyticsStressOptions> {
+    protected final TextAnalyticsAsyncClient textAnalyticsAsyncClient;
+    protected final TextAnalyticsClient textAnalyticsClient;
+
+    /**
+     * Creates an instance of performance test.
+     * @param options the options configured for the test.
+     */
+    public ServiceTest(TextAnalyticsStressOptions options) {
+        super(options);
+        textAnalyticsAsyncClient = new TextAnalyticsClientBuilder()
+            .endpoint(Configuration.getGlobalConfiguration().get("AZURE_TA_ENDPOINT"))
```

Comment:
Should we mention about the required creds to run this test in the readme setup document?

Reply:
nit: Should we use TA abbreviation?